### DISCUSSION TITLE
Security Scans

### DISCUSSION THREAD

**Discussion by jesseg34** [directus/directus#6270] [Archive]
Is directus leveraging any security scanning tools to check the codebase for common vulnerabilities such as SAST or DAST?

**Comment by rijkvanzanten** [comment]
We tried SonarCloud for a bit, but had to disable it as it kept tripping up a lot on the custom syntax things from Vue.. Any recommendations in this field?

**Reply by jesseg34** [reply]
I haven't used it before but [GitHub Advanced Security](https://docs.github.com/en/code-security/secure-coding/automatically-scanning-your-code-for-vulnerabilities-and-errors/about-code-scanning) looks promising. I work mostly in .NET so I don't have any recommendations for JavaScript. I personally think it would be a nice addition to the security policy and help interested organizations with vetting the project.

**Reply by paescuj** [reply]
Some interesting tools:
* https://github.com/returntocorp/semgrep
* https://github.com/ZupIT/horusec
* https://lgtm.com/
